# This is a stamped ros message who implement map
std_msgs/Header header
string[] key
float64[] value